lib/screens/add_exercise_screen.dart

```diff
@@ -46,7 +46,9 @@ class _AddExerciseStepperState extends State<AddExerciseStepper> {
   int _currentStep = 0;
   int lastStepIndex = AddExerciseStepper.STEPS_IN_FORM - 1;
   bool _isLoading = false;
+  bool _isValidating = false;
   Widget errorWidget = const SizedBox.shrink();
+  String? _validationError;
```

Comment:
you can take a look at `FormHttpErrorsWidget`, you can give it a `WgerHttpException` directly and it will render it nicely